src/core/audio/clock.hpp

```diff
@@ -55,6 +62,12 @@ namespace otto::core::clock {
       return (std::max(end, Time{1}) - 1) / n - (std::max(begin, Time{1}) - 1) / n;
     }
 
+    int count_multiple(Time n, float offset) const noexcept
+    {
+      OTTO_ASSERT(n > 0);
+      return (std::max(end - (Time)(n * offset), Time{1}) - 1) / n - (std::max(begin - (Time)(n * offset), Time{1}) - 1) / n;
```

Comment:
And again, this wont work with `unsigned` time. But I guess this means we shouldn't be using that at all, so this code can stay as is